src/styles/index.scss

```diff
@@ -1,3 +1,35 @@
+@import 'vars';
+@import 'header';
+@import 'nav';
+@import 'card';
+@import 'stars';
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: grid;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text-secondary-color;
```

Comment:
The variable `$text-secondary-color` is used here, but it is not defined within this file. Ensure that `$text-secondary-color` is defined in the 'vars' file or another imported file.